migrations/20220402211149-add-place-name-slug.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn(
+        'places',
+        'name_slug',
+        {
+          type: Sequelize.STRING,
+          allowNull: true,
```

Comment:
Is it desired not to have a slug on the place?

Reply:
Good point, I'll change it.